core/src/main/java/bio/terra/pearl/core/dao/participant/FamilyDao.java

```diff
@@ -25,10 +25,19 @@ public Optional<Family> findOneByShortcode(String shortcode) {
         return findByProperty("shortcode", shortcode);
     }
 
+    public List<Family> findByStudyEnvironmentId(UUID studyEnvironmentId) {
+        return findAllByProperty("study_environment_id", studyEnvironmentId);
+    }
+
     public List<Family> findByEnrolleeId(UUID enrolleeId) {
         return jdbi.withHandle(handle -> handle.createQuery("SELECT family.* FROM family family INNER JOIN family_enrollee family_enrollee ON family_enrollee.family_id = family.id WHERE family_enrollee.enrollee_id = :enrolleeId")
                 .bind("enrolleeId", enrolleeId)
                 .mapToBean(Family.class)
                 .list());
     }
+
+    // WARNING: This method is not audited
```

Comment:
can you add an explanation for why? I know why based on the context in this PR, but would be nice to see while perusing code